.github/workflows-src/install-and-test.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2017-2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+name: Test after installing from...
+
+on:
+  push:
```

Comment:
shouldn't we limit CI on to run on pushes to `master` only?

Reply:
We most certainly want this to run on any branch created.